activities.json

```
@@ -2091,5 +2091,17 @@
     "org": "WHATWG",
     "title": "timezonechange event",
     "url": "https://github.com/whatwg/html/pull/3047"
+  },
+  {
+    "ciuName": null,
+    "description": "@scope rule allows targeting CSS rules to subtree or fragment of a document.",
+    "id": "at-scope",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=%40scope",
+    "mozPosition": "positive",
+    "mozPositionDetail": "Scoped styles allow authors to precisely control upper- and lower-bounds of where CSS rules, without having to add many attributes on DOM elements. We hope to evaluate its performance impact by developing a working prototype.",
```

Try to make this second sentence a more direct statement.

```suggestion
    "mozPositionDetail": "Scoped styles allow authors to precisely control upper- and lower-bounds of where CSS rules, without having to add many attributes on DOM elements. However, there is a risk of performance degradation that will have to be answered with implementation experience.",
```